### ISSUE TITLE
Path to all images

### ISSUE THREAD

**Issue by ver0z** [wimlds-trojmiasto/detect-waste#50]
To run train.py of EfficientDet the `path_to_all images` means that I have to take the images of each dataset and join in just one folder ?

python3 train.py path_to_all_images \
--ann_name ../annotations/binary_mixed --model tf_efficientdet_d2 \
--batch-size 4 --decay-rate 0.95 --lr .001 --workers 4 --warmup-epochs 5 \
--model-ema --dataset multi --pretrained --num-classes 1 --color-jitter 0.1 \
--reprob 0.2 --epochs 20 --device cuda:0

**Comment by mariaogryczak** [comment]
Yes, exactly. To use this code without modifications you should put all the images into one folder.
You can modify effdet/data/dataset_config.py and effdet/data/dataset_factory.py dataset classes to add a new dataset and another format of paths for coco annotations type.
For more information see: detect-waste/efficientdet/README.md

**Comment by ver0z** [comment]
Thank you for the response. 

So, as there are dozens of folders in each dataset and it's a little confusing to know which must be the right ones. To be straight to the point could you please let me know which of the folders I have to pick and move to one single folder. I really really thanks for the help.
